sdk/communication/communication-sms/test/smsClientWithToken.spec.ts

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
nit: separating the test utils from the tests looks way cleaner

Reply:
Updated and included a utils folder and new file called recordedClient.ts like the admin folder.